ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -141,6 +141,27 @@ gnomadLink.propTypes = {
   fieldTitle: PropTypes.string,
 }
 
+const GNOMAD_URL_INFO = {
+  urls: { [GENOME_VERSION_37]: 'gnomad.broadinstitute.org', [GENOME_VERSION_38]: 'gnomad.broadinstitute.org' },
+  queryParams: { [GENOME_VERSION_38]: 'dataset=gnomad_r3' },
+}
+
+const sectionTitle = ({ fieldTitle, section }) => (
+  <span>
+    {fieldTitle}
+    &nbsp;
+    {section.toLowerCase()}
+  </span>
+)
+
+sectionTitle.propTypes = {
```

Comment:
functions should not have prop types, react components should. Either make this a react component stylistically by using a capitalized component name, or keep it as a simple function with the current name but remove the prop types

Reply:
Okay. To be consistent in this file, should I also change `gnomadLink`?

Reply:
no thats out of scope